docs/content/en/docs/references/cli/_index.md

```diff
@@ -517,6 +518,19 @@ Env vars:
 * `SKAFFOLD_FORCE` (same as `--force`)
 * `SKAFFOLD_SKIP_BUILD` (same as `--skip-build`)
 
+### skaffold options
+
+
+
+```
+The following options can be passed to any command:
+
+      --color=34: Specify the default output color in ANSI escape codes
```

Comment:
was this autogenerated? where did the `34` come from?

Reply:
lol. This is the default color: blue

Reply:
lol, maybe `--color=blue` would be a little bit more intuitive :)